src/sentry/issues/endpoints/group_similar_issues.py

```diff
@@ -56,9 +56,7 @@ def get(self, request: Request, group: Group) -> Response:
         for group_id, scores in zip(group_ids, group_scores):
             serialized_group = serialized_groups.get(group_id)
             if serialized_group is None:
-                # TODO(tkaemming): This should log when we filter out a group that is
-                # unable to be retrieved from the database. (This will soon be
-                # unexpected behavior, but still possible.)
+                logger.warning(f"Group {group_id} unable to be retrieved from the database.")
```

Comment:
Can you please take this change? I cannot commit unto your branch

```suggestion
                logger.warning(f"Group {group_id} unable to be retrieved from the database.")
                logger.warning("Group %s unable to be retrieved from the database.", group_id)
```